test/assertions.js

```diff
@@ -0,0 +1,142 @@
+const assert = require('assert');
```

Comment:
This file is mostly extracted from expresso, except `assert.response` which actually started a couple hundred http servers at once. (surprise, it didn't work) So instead we use `supertest`.